src/meta/override_values.yaml

```diff
@@ -0,0 +1,22 @@
+# Missing roots due to circular references:
+minecraft:raw_iron: 100
+minecraft:iron_ingot: 100
+minecraft:coal: 20
+minecraft:raw_gold: 200
+minecraft:gold_ingot: 200
+minecraft:raw_copper: 50
+minecraft:copper_ingot: 50
+minecraft:diamond: -1
+minecraft:emerald: 200
+minecraft:lapis_lazuli: 50
+minecraft:redstone: 50
```

Comment:
Also too high. I have like 8 stacks of redstone, mined during a single trip. That would give me a total value of 25600 here. Basically 5 elytras